src/pages/index.tsx

```diff
@@ -10,16 +13,44 @@ import {
 import { type NextPage } from 'next';
 import AddCharacterForm from '../components/AddCharacterForm';
 
+import SearchBar from '../components/SearchBar';
 import { api } from '../utils/api';
 
 const Home: NextPage = () => {
   const characters = api.characterRouter.getCharacters.useQuery()?.data;
 
   return (
     <Center minH="100vh" backgroundColor="gray.900">
```

Comment:
We can change the background color here (we should also be using `bgColor` instead of `backgroundColor`—that's my bad) so that when scrolling down past the length of the image, the color doesn't abruptly change to gray.

Reply:
changed (would be better to add these colors to the styles later instead)